2023/articles/2023-12-01.pod

```diff
@@ -0,0 +1,71 @@
+Title: Arango::Tango, the new mammal in town
+Topic: Arango::Tango
+Author: Alberto Simões <ambs@cpan.org>
+
+=encoding utf8
+
+=pod
+
+L<Arango::Tango> is not properly new, as the first versions are from 2019. Nevertheless, the number of users is still quite small as 
+other document-oriented databases, like L<ElasticSearch|https://www.elastic.co/> or C<MongoDB|https://www.mongodb.com/> are still quite
+popular. But beging somehow alergic to Java, I went in the search for an alternative and found C<ArangoDB|https://arangodb.com/>, a graph
+database (also able to deal with plain document collections) written in C++.
```

Comment:
Maybe a couple of sentences on what a graph database is and why you'd use such a thing.

Reply:
Sure.